### PR TITLE
DigiDNA April 2021 tweaks

### PR DESCRIPTION
**Pull request by relgit**
This PR contains two housekeeping items:

- Redundant keys removed in certain property dictionaries
- `ManagedFavorites` structure updated in Chromium-based browser manifests

### CODE REVIEW

**Review by apizz — APPROVED** [review]
All look good to me.  Thanks for adding / fixing the toplevel_name for the bookmark folder

File: Manifests/ManagedPreferencesApplications/com.microsoft.Edge.plist [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	<key>pfm_interaction</key>
 	<string>combined</string>
 	<key>pfm_last_modified</key>
-	<date>2020-11-21T16:33:42Z</date>
+	<date>2021-04-01T09:24:39Z</date>
```
Comment (apizz) [inline Manifests/ManagedPreferencesApplications/com.microsoft.Edge.plist:18]: Should bump the pfm_version of Edge like you did for Chrome & Brave

**Review by apizz — CHANGES_REQUESTED** [review]
Sorry, missed that Edge did not have a pfm_version bump.  That one change should be good to merge

File: Manifests/ManagedPreferencesApplications/com.microsoft.Edge.plist [hunk 2]
```diff
@@ -5707,8 +5707,15 @@ Managed bookmarks are not synced to the user account and can't be modified by ex
 					<string>77</string>
 					<key>pfm_description</key>
 					<string>The top-level managed favorites folder name.</string>
-					<key>pfm_name</key>
-					<string>toplevel_name</string>
+					<key>pfm_subkeys</key>
+					<array>
+						<dict>
+							<key>pfm_name</key>
+							<string>toplevel_name</string>
```
Comment (apizz) [inline Manifests/ManagedPreferencesApplications/com.microsoft.Edge.plist:5714]: Can we add a more user-friendly `pfm_title` for this?

**Review by apizz — CHANGES_REQUESTED** [review]
Very small tweak

**Comment by apizz** [comment]
Need to address the conflicting files

**Comment by relgit** [comment]
> Need to address the conflicting files

Done.

**Pull request merged**